modules/core/04-channel/types/packet.go

```diff
@@ -101,42 +101,6 @@ func commitV2Packet(packet Packet) []byte {
 	return hash[:]
 }
 
-// CommitPacketV2 returns the V2 packet commitment bytes. The commitment consists of:
```

Comment:
These methods are not used right now and can be reimplemented in the right place (they wouldn't compile left here now) in further PRs

Reply:
You could probably just move them to the v2/types directory in a new packet.go file. What do you think? Might be easier to keep them rather than digging into old/historical branches to find as we move forward.

These should ultimately change eventually anyways pending spec

Reply:
Yeah SGTM :) Will do! (I will still get rid of the conversion function that we don't need anymore at all)